Rakefile

```diff
@@ -13,6 +12,24 @@ else
   task :default => :spec
 end
 
+task :appraisal do
+  for_all_gemfiles("exec", "rake")
+end
+
+namespace :appraisal do
+  task :install do
+    for_all_gemfiles("install")
+  end
+end
+
+def for_all_gemfiles(*args)
+  Dir.glob("gemfiles/rails_{5.2,6.0,edge}/Gemfile").each do |gemfile|
+    Bundler.with_original_env do
```

Comment:
Rdoc says `with_original_env` is deprecated in favour of `with_unbundled_env`.

Reply:
Mmmm, tell me where please because I should fix that since it's incorrect :grimacing:. `Bundler.with_clean_env` is the one deprecated in favor of `Bundler.with_unbundled_env`.

Reply:
Also, there is `unbundled_exec` in latest bundler.

Reply:
My bad, confused those two :)